### PR TITLE
Temporarily hack a non interactive solution for setup:account command so Salto will be able to use it directly using TS

### PR DESCRIPTION
**Pull request by omrilit**

### CODE REVIEW

**Comment by hadard** [comment]
Following our f2f talk, I think this PR changes will make it harder to rebase. Most likely it will take time until this ability will be exposed in the upstream.  I think that your suggestion of adding new non-interactive command will be better.

**Comment by omrilit** [comment]
Done

File: .gitignore [hunk 1]
```diff
@@ -5,8 +5,5 @@
 **/.vscode/**
 **/.idea/**
 
-# package-lock of packages doesn't make sense with Lerna
-**/package-lock.json
```
Comment (hadard) [inline .gitignore]: nit: why removed?

File: packages/node-cli/src/metadata/NodeCommandsMetadata.json [hunk 2]
```diff
@@ -4,14 +4,53 @@
         "alias": "sa",
         "description": "Sets up an account to use with the SuiteCloud CLI for Node.js.",
         "isSetupRequired": false,
-        "forceInteractiveMode": true,
```
Comment (hadard) [inline packages/node-cli/src/metadata/NodeCommandsMetadata.json:7]: now that we have new commands removal of `forceInteractiveMode` is still needed? what about the options below(`authid`, `accountid`, etc.) ?

**Review by hadard — APPROVED** [review]
LGTM marking as approved to save cycles. Please see the comment and the question below.

File: .gitignore (hunk 1 above)
Comment (omrilit) [reply .gitignore]: Actually this one is already committed to master, but appears also here so I'll just rebase.
Anyway the reason for it is that I had to commit the package-lock.json files in order to build salto with this dependency:

> lerna ERR! npm ci --no-package-lock exited 1 in '@oracle/suitecloud-cli'
lerna ERR! npm ci --no-package-lock stderr:
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.

File: packages/node-cli/src/metadata/NodeCommandsMetadata.json (hunk 2 above)
Comment (omrilit) [reply packages/node-cli/src/metadata/NodeCommandsMetadata.json:7]: This is not really a new command.
In the `CommandGenerators.json`, I added a new option for `nonInteractiveGenerator` so both interactive and non-interactive use the same command but with different options and executors.

**Pull request merged**